.github/workflows/unit_tests.yml

```diff
@@ -69,15 +69,10 @@ jobs:
       - name: Install dependencies
         run: make install-python-ci-dependencies
       - name: Test Python
-        env:
-          SNOWFLAKE_CI_DEPLOYMENT: ${{ secrets.SNOWFLAKE_CI_DEPLOYMENT }}
-          SNOWFLAKE_CI_USER: ${{ secrets.SNOWFLAKE_CI_USER }}
-          SNOWFLAKE_CI_PASSWORD: ${{ secrets.SNOWFLAKE_CI_PASSWORD }}
-          SNOWFLAKE_CI_ROLE: ${{ secrets.SNOWFLAKE_CI_ROLE }}
-          SNOWFLAKE_CI_WAREHOUSE: ${{ secrets.SNOWFLAKE_CI_WAREHOUSE }}
         run: pytest -n 8 --cov=./ --cov-report=xml --color=yes sdk/python/tests
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v1
+        if: github.repository == 'feast-dev/feast'
```

Comment:
does this then fail if the tests break? i think the action may always succeed even if tests fail

Reply:
Hm I don't think so? The unit tests were failing on my fork.